### PR TITLE
WIP: Attempt to fix the test bug

### PR DESCRIPTION
**Pull request by springcoil**
This is a really trivial attempt to fix the test bug - no idea if it'll work or not.

### CODE REVIEW

**Comment by springcoil** [comment]
This passes the tests - so I guess it's ok to merge. Any views @twiecki?

**Comment by twiecki** [comment]
Can we move it inside the test itself?

File: pymc3/tests/test_distributions_random.py [hunk 1]
```diff
@@ -530,7 +530,7 @@ def test_categorical(self):
             'Categorical cannot be initialised with >1D probabilities')
         self.check(Categorical, p=self.ones / self.n)
 
-
+nr.seed(20090425)
 @attr('scalar_parameter_samples')
 class ScalarParameterSamples(unittest.TestCase):
```
Comment (springcoil) [inline pymc3/tests/test_distributions_random.py:535]: You mean in below class ScalarParameterSamples cc @twieki

Comment (twiecki) [reply pymc3/tests/test_distributions_random.py:535]: Yeah.

Comment (twiecki) [reply pymc3/tests/test_distributions_random.py:535]: Although I guess this is fine as is.

**Comment by springcoil** [comment]
Ohh you merged it - I was trying to fix that :) It's fine.

**Pull request merged**